### PR TITLE
Add ingress node firewall daemon image

### PR DESCRIPTION
**Pull request by msherif1234**
include ds image build
Signed-off-by: Mohamed Mahmoud <mmahmoud@redhat.com>

### CODE REVIEW

**Comment by msherif1234** [comment]
https://github.com/openshift/ingress-node-firewall/pull/94 hopefully will fix make lint and make unit-test

File: ci-operator/config/openshift/ingress-node-firewall/openshift-ingress-node-firewall-master.yaml [hunk 1]
```diff
@@ -34,13 +41,11 @@ tag_specification:
...
   namespace: ocp
 tests:
-- as: e2e-aws
-  steps:
-    cluster_profile: aws
-    workflow: openshift-e2e-aws
-- as: ingress-node-firewall-e2e-metal-ipi
-  steps:
-    cluster_profile: packet
+- as: test-fmt
```
Comment (dhellmann) [inline ci-operator/config/openshift/ingress-node-firewall/openshift-ingress-node-firewall-master.yaml:44]: Is this replacing e2e tests with linter tests?

Comment (msherif1234) [reply ci-operator/config/openshift/ingress-node-firewall/openshift-ingress-node-firewall-master.yaml:44]: we are working on bringing in the e2e test we have today on kind to run here in separate PRs till we have all OCP's bits in place

**Comment by msherif1234** [comment]
here is the EPIC Jira https://issues.redhat.com/browse/SDN-3371

**Comment by dhellmann** [comment]
This looks OK. I know you've been working with @jupierce so I want to give him a chance to ack the name and other changes before we go ahead.

**Pull request merged**